### DISCUSSION TITLE
WiFi Events

### DISCUSSION THREAD

**Discussion by i-g-g-y** [khoih-prog/ESP_WiFiManager_Lite#17] [Q&A]
Hello. 

I am using WiFi Manager Lite with Async MQTT and I am trying to get an event driven activity going but the WiFi events do not get called?

I am using station handlers and not one event gets called, what could be the reason?

These are the handlers I want to use:
```
  // Call "onStationConnected" each time a station connects  
  stationConnectedHandler = WiFi.onStationModeConnected(&onStationConnected);  
  
  // Call "onStationDisconnected" each time a station disconnects  
  stationDisconnectedHandler = WiFi.onStationModeDisconnected(&onStationDisconnected);

  stationGotIPHandler = WiFi.onStationModeGotIP(&onStationGotIP);
```

**Comment by khoih-prog** [comment]
Hi @i-g-g-y 

Without the full code, or at least the [**MRE**](https://stackoverflow.com/help/minimal-reproducible-example), I don't think anybody can know what's wrong with your code, therefore the reason to help you out.

You can also post on Arduino Forum to ask for help, certainly with [**MRE**](https://stackoverflow.com/help/minimal-reproducible-example).

**Reply by i-g-g-y** [reply]
Hello, 

thanks for answer, it was not my intent for you to support my development, but many thanks for offering. 

The question is if there is something in your code, that could prevent those handlers from working , or if you know of any other reason the handlers would not work, but in connection to your library?
